spec/truefi2/BorrowingMutex.spec

```diff
@@ -34,6 +34,8 @@ rule functionDoesNotBanUnlockedBorrower() {
         address locker = anyReasonableAddress();
```

Comment:
https://prover.certora.com/output/67403/aebc1e73f163fd7941bc?anonymousKey=3923171e43a2f72efa42f42cc27897ebb8965c7a

Reply:
👍 

Once #1136 gets checked in to main, we won't need to manually link this any more.